sdk/src/main/java/com/bugsnag/android/AppData.java

```diff
@@ -15,15 +10,7 @@
  */
 public class AppData extends AppDataSummary {
 
-    private static final long startTimeMs = SystemClock.elapsedRealtime();
-
-    @Nullable
-    final String appName;
-
-    @NonNull
-    private final Context appContext;
-    private final SessionTracker sessionTracker;
-
+    @SuppressWarnings("NullableProblems") // initialised after construction
```

Comment:
Warnings are suppressed as we're anticipating that this field will always be set to a non-null value after construction. This will allow users to access the value in a Kotlin-esque way, i.e.:

`appData.packageName`

Rather than requiring the safe call operator each time to unwrap the optional:

`appData?.packageName`